### DISCUSSION TITLE
Question about Snapper and Nala

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#251] [Q&A] [answered]
Can I use `nala` instead of `apt`? Will Snapper still make system snapshots automatically if I use Nala? Or is this unsupported?

**Comment by robertocannito** [comment, marked as answer]
I've installed but I've seen that with btrfs-assistant it takes two pre snapshots instead on one.

I don't know without it and only with snapper gui works good

you should try, I suggest first in virtual machine and see the what happen

**Reply by birds-swim** [reply]
I will try that out.  Thank you for your response!
